Makefile

```
@@ -259,6 +259,9 @@ lint:
 		bash -c "[[ ! -z '$${SEARCH_PATH}' ]] && find $${SEARCH_PATH} -name '*.py' | sort -u | xargs pylint --py3k" && \
 		echo "--- Linting done. ---"; \
 	fi
+	. $(VENVNAME)/bin/activate; \
+	git diff $(MASTER_BRANCH)..HEAD --name-only | xargs isort -c --diff && \
+	echo "--- isort check done. ---;"
```

I like where this is going - here's an untested suggestion that will automatically apply the generated patch using `git apply` and then use `git diff` to print the changes.

```suggestion
	git diff $(MASTER_BRANCH)..HEAD --name-only | xargs isort -c --diff | git apply - && git diff --exit-code && \
	echo "--- isort check done. ---;" || echo "--- isort check done - new changes staged. ---;"
```

---

Well hypothetically isort auto-applied changes can introduce syntax errors, so I would leave the actual fixing up to the authors (they can run isort themselves and verify the changes). I'd leave linter it's usual role by default - inform about the problem, not fix it.

---

Added a lint_fix target.
Also switched from git diff master..HEAD to git diff master this should catch problems even if files that are changed but not committed yet.